Thesis.tex

```diff
@@ -222,11 +224,65 @@ \subsection{Reasonable and non-discriminatory licensing}
  
 Another important case related to RAND is Microsoft versus Motorola. In 2012, the United States Court of Appeals for the Ninth Circuit ruled that a patent holder's agreement to a policy like RAND creates a legally binding contract \parencite{MicrosoftVsMotorola}.
 
+\chapter{Discourse}
+\label{ch:Disc}
+
+\section{Copyright in the context of the arts}
+\label{sec:CopyArts}
+
+\subsection{Pro IP}
+In \cite{Towse1999} it's highlighted that the production of a work, conceiving ideas, ordering them and structuring them to a finished work, takes time and energy. To incentivise this investment of energy is the main goal of copyright. In granting the creator of a work the right to control and restrict the acts of copying (for example copies, but also performances and broadcasts of said work), it ensures that future works can be produced.
+
+It also highlights the costs necessitated for the production of a sound recording. This initial cost is not needed by a reproducer of the same work. Copyright thus also protects the publisher, which usually fronts the cost as an investment, from undercutting by competitors which do not have to carry that initial cost of producing a record. The transfer of copyright from creator to publisher, can be organised in a multitude of schemes. The most prominent ones are royalty (in which the creator gets a share from the sale of the work) and 'buy-out' arrangements (in which the creator gets one payment for all his rights), and Towse does not fail to highlight the skewed power dynamic between these two parties (publishers have more possibility to distribute risks through a bigger portfolio of individual creators, compared to creators who only have their work and own human capital).
```

Comment:
Copyright thus also protects the publisher, which usually fronts the cost as an investment, from **being undercut** by competitors which do not have to carry that initial cost of producing a record.